### PR TITLE
docs: update readme with cron and cronjob information

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR updates the readme description and add information about the how to run the scheduled tasks for email digest in the `Deploying` section.

**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
 ####################################
 
-.. note::
-
-  This README was auto-generated. Maintainer: please review its contents and
-  update all relevant sections. Instructions to you are marked with
-  "PLACEHOLDER" or "TODO". Update or remove those sections, and remove this
-  note when you are done.
 
 |pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
```
Comment (BryanttV) [inline README.rst]: We should only leave the labels that apply

File: README.rst
Comment (BryanttV) [inline README.rst:1]: We should use the `security@edunext.co` email in the reporting of security issues.

Comment (Ian2012) [reply README.rst:1]: fixed

File: README.rst [hunk 2]
```diff
@@ -160,34 +130,24 @@ file in this repo.
...
+Please do not report security issues in public. Please email security@edunext.co.
 
-.. |pypi-badge| image:: https://img.shields.io/pypi/v/platform_plugin_forum_email_notifier.svg
+.. It's not required by our contractor at the moment but can be published later
+.. .. |pypi-badge| image:: https://img.shields.io/pypi/v/platform_plugin_forum_email_notifier.svg
     :target: https://pypi.python.org/pypi/platform_plugin_forum_email_notifier/
     :alt: PyPI
 
 .. |ci-badge| image:: https://github.com/eduNEXT/platform_plugin_forum_email_notifier/workflows/Python%20CI/badge.svg?branch=main
     :target: https://github.com/eduNEXT/platform_plugin_forum_email_notifier/actions
```
Comment (BryanttV) [inline README.rst]:
```suggestion
    :target: https://github.com/eduNEXT/platform-plugin-forum-email-notifier/actions
```

File: README.rst [hunk 3]
```diff
@@ -160,34 +130,24 @@ file in this repo.
...
-.. |doc-badge| image:: https://readthedocs.org/projects/platform_plugin_forum_email_notifier/badge/?version=latest
-    :target: https://docs.openedx.org/projects/platform_plugin_forum_email_notifier
-    :alt: Documentation
-
-.. |pyversions-badge| image:: https://img.shields.io/pypi/pyversions/platform_plugin_forum_email_notifier.svg
-    :target: https://pypi.python.org/pypi/platform_plugin_forum_email_notifier/
-    :alt: Supported Python versions
-
 .. |license-badge| image:: https://img.shields.io/github/license/eduNEXT/platform_plugin_forum_email_notifier.svg
     :target: https://github.com/eduNEXT/platform_plugin_forum_email_notifier/blob/main/LICENSE.txt
```
Comment (BryanttV) [inline README.rst]:
```suggestion
    :target: https://github.com/eduNEXT/platform-plugin-forum-email-notifier/blob/main/LICENSE.txt
```

File: README.rst [hunk 4]
```diff
@@ -80,45 +70,25 @@ Every time you develop something in this repo
...
-
-Getting Help
-************
+The email digest feature works by acumulating the notifications in a database table
+per user, per course and per digest frequency. Then, a scheduled task is run to
+send the notifications to the users.
 
-Documentation
-=============
+As Open edX doesn't support celery beat for scheduled tasks, so we need to use
```
Comment (mariajgrimaldi) [inline README.rst]:
```suggestion
As Open edX doesn't support celery beat for scheduled tasks, we need to use
```

File: README.rst [hunk 5]
```diff
@@ -80,45 +70,25 @@ Every time you develop something in this repo
...
-
-PLACEHOLDER: For details on how to deploy this component, see the `deployment how-to`_
-
-.. _deployment how-to: https://docs.openedx.org/projects/platform_plugin_forum_email_notifier/how-tos/how-to-deploy-this-component.html
-
-Getting Help
-************
+The email digest feature works by acumulating the notifications in a database table
+per user, per course and per digest frequency. Then, a scheduled task is run to
+send the notifications to the users.
```
Comment (mariajgrimaldi) [inline README.rst]:
```suggestion
The email digest feature accumulates the notifications in a database table
per user, per course, and digest frequency. Then, a scheduled task is run to
send the notifications to the users.
```

File: README.rst [hunk 6]
```diff
@@ -80,45 +70,25 @@ Every time you develop something in this repo
...
 
-PLACEHOLDER: Start by going through `the documentation`_.  If you need more help see below.
-
-.. _the documentation: https://docs.openedx.org/projects/platform_plugin_forum_email_notifier
-
-(TODO: `Set up documentation <https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs>`_)
-
-More Help
-=========
+For tutor local installations we need to use cron to run the scheduled tasks.
```
Comment (mariajgrimaldi) [inline README.rst]: cron? can we add a link to the documentation?

File: tutor-plugins/forum_digest.yml [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+#   `forum-email-notifier-daily-digest-cronjob`: send daily digest emails.
+#   `forum-email-notifier-weekly-digest-cronjob`: send weekly digest emails.
+# Note: The actual digest is triggered depending on the `last_sent` field of the
+# model, and do not depend on the schedule of the job. The job is scheduled
+# to run at midnight to avoid any timezone issue.
+name: forum-email-notifier
+patches:
+  k8s-deployments: |
+    ---
+    # Run daily forum email notifier
```
Comment (mariajgrimaldi) [inline tutor-plugins/forum_digest.yml]:
```suggestion
    # Run daily forum email notifications
```

File: tutor-plugins/forum_digest.yml [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+                  configMap:
+                    name: openedx-config
+                - name: uwsgi-config
+                  configMap:
+                    name: openedx-uwsgi-config
+                    items:
+                    - key: uwsgi.ini
+                      path: uwsgi.ini
+    ---
+    # Run weekly forum email notifier
```
Comment (mariajgrimaldi) [inline tutor-plugins/forum_digest.yml]: # Run weekly forum email notifications

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**